cannot install

when I run ```make install``` the next error appears

```
~/maglev(branch:master) » make install
mv ~/.tmux.conf ~/.tmux.conf.bak
cp `pwd`/.tmux.conf ~/.tmux.conf
sed -i '' 's|{{pwd}}|'`pwd`'|g' ~/.tmux.conf
sed: can't read s|{{pwd}}|/home/phi/maglev|g: No such file or directory
Makefile:2: recipe for target 'install' failed
make: *** [install] Error 2
```

I still got the same error

```
mv ~/.tmux.conf ~/.tmux.conf.bak
cp `pwd`/.tmux.conf ~/.tmux.conf
sed -i '' 's|{{pwd}}|'`pwd`'|g' ~/.tmux.conf
sed: can't read s|{{pwd}}|/home/oscar/com.github/maglev|g: No such file or directory
Makefile:2: recipe for target 'install' failed
make: *** [install] Error 2
```

I got the same problem when I do
```
make install
```
the return is
```
...
Resolving deltas: 100% (151/151), done.
Submodule path 'plugins/yank': checked out '13a82191193d54959153a19e4c6b674c7f29ab75'
cp `pwd`/.tmux.conf ~/.tmux.conf
sed -i '' 's|{{pwd}}|'`pwd`'|g' ~/.tmux.conf
sed: can't read s|{{pwd}}|/home/****/maglev|g: No such file or directory
make: *** [install] Error 2
```
OS is RHEL7

I have the same problem on Debian Jessie.

```
sed -i '' 's|{{pwd}}|'`pwd`'|g' ~/.tmux.conf
sed: can't read s|{{pwd}}|/home/jurre/dev/maglev|g: No such file or directory
Makefile:2: recipe for target 'install' failed
make: *** [install] Error 2
```

i have same problem in ubuntu 15.04

```
make install
git submodule init
git submodule update
mv ~/.tmux.conf ~/.tmux.conf.bak
cp `pwd`/.tmux.conf ~/.tmux.conf
sed -i '' 's|{{pwd}}|'`pwd`'|g' ~/.tmux.conf
sed: can't read s|{{pwd}}|/media/data/Software/maglev|g: No such file or directory
Makefile:2: recipe for target 'install' failed
make: *** [install] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -1,3 +1,4 @@
+opsys := $(shell sh -c 'uname -s 2>/dev/null || echo not')
 install:
 	git submodule init
 	git submodule update
@@ -7,7 +8,12 @@ install:
 		mv ~/.tmux.conf ~/.tmux.conf.bak; \
 	fi;
 	cp `pwd`/.tmux.conf ~/.tmux.conf
-	sed -i '' 's|{{pwd}}|'`pwd`'|g' ~/.tmux.conf
+	@if [ "$(opsys)" = "Linux" ]; \
+	then \
+		sed -i 's|{{pwd}}|'`pwd`'|g' ~/.tmux.conf; \
+	@else \
+		sed -i '' 's|{{pwd}}|'`pwd`'|g' ~/.tmux.conf; \
+	fi \
 
 uninstall:
 	@if [ -f ~/.tmux.conf.bak ]; \

